cmd/git-sync/main.go

```diff
@@ -286,6 +326,10 @@ func main() {
 	for {
 		start := time.Now()
 		ctx, cancel := context.WithTimeout(context.Background(), time.Second*time.Duration(*flSyncTimeout))
+		if *flSpecific {
+			h, m := sanitizeSpecificTime(*flSpecificTime)
```

Comment:
do this outside the for loop. We dont need to resanitize every time we run the for loop. The flag will not change